media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -10,30 +10,33 @@ than originally without affecting the sound pitch.
 tempo (speed).
 Playback Rate : Changes both tempo and pitch together as if a vinyl disc was \
 played at different RPM rate."
-HOMEPAGE="http://www.surina.net/soundtouch"
-COPYRIGHT="2001-2014 Olli Parviainen"
+HOMEPAGE="https://www.surina.net/soundtouch/"
+COPYRIGHT="2001-2018 Olli Parviainen"
 LICENSE="GNU LGPL v2.1"
 REVISION="1"
-SOURCE_URI="http://www.surina.net/soundtouch/soundtouch-$portVersion.tar.gz"
-CHECKSUM_SHA256="d224f7d1421b5f8e74a74c85741345bd9802618a40ae30ce5b427a5705c89d25"
-SOURCE_DIR="soundtouch"
-PATCHES="libsoundtouch-$portVersion.patchset"
+SOURCE_URI="https://gitlab.com/soundtouch/soundtouch/-/archive/$portVersion/soundtouch-$portVersion.tar.bz2"
+CHECKSUM_SHA256="3fb740c7ac4b304342fbf11a91d14fa3e296269b9fec6266609c4620477d0ad6"
+SOURCE_DIR="soundtouch-$portVersion"
+
+libVersion="1.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= 1"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	libsoundtouch$secondaryArchSuffix = $portVersion compat >= 1
-	cmd:soundstretch$secondaryArchSuffix
-	lib:libSoundTouch$secondaryArchSuffix = 1.0.0 compat >= 1
+	libsoundtouch$secondaryArchSuffix = $portVersionCompat
+	cmd:soundstretch$secondaryArchSuffix = 2.1
```

Comment:
```diff
-	cmd:soundstretch$secondaryArchSuffix = 2.1
+	cmd:soundstretch$secondaryArchSuffix = $portVersion
```
(or maybe `$portVersionCompat` instead of `$portVersion`, or `$portVersion compat >= 2.1`)

Hmm, BTW, if we're sure that the program cannot be built with gcc2 then it might be a good idea to define a [`commandBinDir`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable) and `commandSuffix`.

Reply:
Already changed locally for the portVersion
libsoundtouch failing for gcc2 with "cpu_detect_x86.cpp:77: Invalid token in expression"

Reply:
> Already changed locally for the portVersion

Oops, we still have `2.1`, while `$portVersion` is `2.1.0`.

Reply:
Now we don't :)